app.rb

```diff
@@ -0,0 +1,152 @@
+# app.rb
+require_relative 'library'
+
+class App
+  def initialize
+    @library = Library.new
+  end
+
+  def main
+    puts 'Welcome to the Library Management System!'
+    loop do
+      print_options
+      option = gets.chomp.to_i
+      break if option == 7
+
+      execute_option(option)
+    end
+    puts 'Thank you for using the Library Management System. Goodbye!'
+  end
+
+  private
+
+  def print_options
+    puts "\nSelect an option:"
+    puts '1. List all books.'
+    puts '2. List all people.'
+    puts '3. Create a teacher.'
+    puts '4. Create a student.'
+    puts '5. Create a book.'
+    puts '6. Create a rental.'
+    puts '7. Quit.'
+  end
+
+  def execute_option(option)
+    case option
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_teacher
+    when 4
+      create_student
```

Comment:
- [ ] Both create teacher and create student are under create person and not separated, the fact that you separated it makes no space for list rentals but only create rentals. I will implore you to add it.

### Expected UI
![image](https://github.com/mpa-mxiang/school_library/assets/86200356/a6ebc50b-b225-43d6-8677-47e6500ed989)

### Your UI
![image](https://github.com/mpa-mxiang/school_library/assets/86200356/0d370a10-be44-4462-ae98-b9d106376759)